main.go

```diff
@@ -81,5 +88,46 @@ func main() {
 		}
 
 		sendEmail(to, cc, from, subject, body, username, password)
+
+	case "calendar":
+		driveService, err, transport := newDriveService()
+
+		if err != nil {
+			log.Fatalf("Can not created drive service: %v", err)
+		}
+
+		if _, err := os.Stat(eventFilePath); !os.IsNotExist(err) {
+			println("Old file is deleting")
+			os.Remove(eventFilePath)
+		}
+
+		downloadFile(driveService, transport, &drive.File{
+			DownloadUrl:"https://docs.google.com/spreadsheets/export?id=1URxws-2gXrAHQ0vUPQxcFIswK9VM5NcdEwaxZaOZ9W4&exportFormat=xlsx",
+		})
+
+		calendarService, err := newCalendarService()
+
+		if err != nil {
+			log.Fatalf("Can not created calendar service: %v", err)
```

Comment:
"Can NOT instantiate Google Calendar Service Client"

ve bir de bunun gerceklesemiyor olmasi gercekten `FATAL` bir durum mudur? Buna karar verdin mi?